src/Evolution/Systems/GeneralizedHarmonic/DampedHarmonicGaugeQuantities.hpp

```diff
@@ -0,0 +1,435 @@
+tnsr::a<DataType, SpatialDim, Frame> spacetime_deriv_of_log_factor(
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift,
+    const tnsr::A<DataType, SpatialDim, Frame>& spacetime_unit_normal,
+    const tnsr::II<DataType, SpatialDim, Frame>& inverse_spatial_metric,
+    const Scalar<DataType>& sqrt_det_spatial_metric,
+    const tnsr::ii<DataType, SpatialDim, Frame>& dt_spatial_metric,
+    const tnsr::aa<DataType, SpatialDim, Frame>& pi,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
+    double exponent) noexcept;
+// @}
+
+// @{
+/*!
+ * \ingroup GeneralizedHarmonic
+ * \brief Computes spacetime derivatives of the log factor (that appears in
+ * damped harmonic gauge source function) raised to an exponent.
+ *
+ * \details Computes the spacetime derivatives:
+ * \f[
+ *  \partial_a (logF)^q = q (logF)^{q-1} \partial_a (logF)
+ * \f]
+ */
+template <size_t SpatialDim, typename Frame, typename DataType>
+void spacetime_deriv_of_power_log_factor(
+    gsl::not_null<tnsr::a<DataType, SpatialDim, Frame>*> d4_powlogfac,
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift,
+    const tnsr::A<DataType, SpatialDim, Frame>& spacetime_unit_normal,
+    const tnsr::II<DataType, SpatialDim, Frame>& inverse_spatial_metric,
+    const Scalar<DataType>& sqrt_det_spatial_metric,
+    const tnsr::ii<DataType, SpatialDim, Frame>& dt_spatial_metric,
+    const tnsr::aa<DataType, SpatialDim, Frame>& pi,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi, double g_exponent,
+    int exponent) noexcept;
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::a<DataType, SpatialDim, Frame> spacetime_deriv_of_power_log_factor(
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift,
+    const tnsr::A<DataType, SpatialDim, Frame>& spacetime_unit_normal,
+    const tnsr::II<DataType, SpatialDim, Frame>& inverse_spatial_metric,
+    const Scalar<DataType>& sqrt_det_spatial_metric,
+    const tnsr::ii<DataType, SpatialDim, Frame>& dt_spatial_metric,
+    const tnsr::aa<DataType, SpatialDim, Frame>& pi,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi, double g_exponent,
+    int exponent) noexcept;
+// @}
+}  // namespace DampedHarmonicGauge_detail
+
+// @{
+/*!
+ * \ingroup GeneralizedHarmonic
+ * \brief Damped harmonic gauge source function.
+ *
+ * \details Computes the gauge source function designed for binary black hole
+ * evolutions. These have been taken from "Simulations of Binary Black Hole
+ * Mergers Using Spectral Methods" by Szilagyi, Lindblom and
+ * Scheel \cite Szilagyi2009qz, and "Critical behavior in 3D gravitational
+ * collapse of massless scalar fields" by Deppe+ \cite Deppe2018uye.
+ *
+ * The covariant form of the source function \f$H_a\f$ is written as:
+ *
+ * \f{align*}
+ * H_a :=  A_{H_\mathrm{init}} [1 - R_{H_\mathrm{init}}(t)] H_a^\mathrm{init} +
```

Comment:
Why do you use the amplitude `A_{H_\mathrm{init}}` here? It is not present in the references and I doubt, that we will ever need it.

Reply:
We have amplitude control switches `A_L1, A_L2, A_S` for every term, and I added `A_{H_\mathrm{init}}` for the initial gauge for future convenience and symmetry. Is there any harm in leaving it in?

Reply:
It takes a small amount of unnecessary additional computation.

Reply:
removed `A_{H_\mathrm{init}}`